rabbitmq-http-connector/README.md

```diff
@@ -0,0 +1,14 @@
+Kafka KEDA connector image can be used in the Kubernetes deployment as scaleTargetRef in scaledObject of [RabbitMQ scaler](https://keda.sh/docs/1.5/scalers/rabbitmq-queue/).
+
+The job of the connector is to read messages from the queue, request an endpoint, and write response or error in the respective queue. Following enviornment variables are used by connector image as configuration to connect and authenticate with RabbitMQ cluster which should be defined in the Kubernetes deployment manifest.
```

Comment:
Instead of `queue, request an endpoint, and` I would write `queue, call an HTTP endpoint with the body of the message, and`

Reply:
Yes, that sounds better. I have updated same.